velox/exec/tests/OrderByTest.cpp

```diff
@@ -463,6 +463,12 @@ TEST_F(OrderByTest, spill) {
   EXPECT_LT(0, stats[0].operatorStats[1].spilledBytes);
   EXPECT_EQ(1, stats[0].operatorStats[1].spilledPartitions);
   EXPECT_EQ(2, stats[0].operatorStats[1].spilledFiles);
+  // stats[0].operatorStats[1] is OrderBy operator and stats[0].operatorStats[0]
+  // is the parent operator of OrderBy. And the finishTiming.wallNanos in
+  // OrderBy operator  is greater than the parent's.
+  EXPECT_GT(
```

Comment:
Would this test fail when without this PR?

Reply:
Yes. The test will fail withour this PR.